readthedocs/subscriptions/notifications.py

```diff
@@ -93,7 +93,7 @@ def for_organizations(cls):
         body=_(
             textwrap.dedent(
                 """
-            The organization "{{instance.slug}}" is currently disabled. You need to <a href="{{subscription_url}}">renew your subscription</a> to keep using Read the Docs
+            The organization "{{instance.slug}}" is currently disabled. You need to <a href="{% url 'subscription_detail' instance.slug %}">renew your subscription</a> to keep using Read the Docs
```

Comment:
Don't we need to also remove where we are passing this value?

Reply:
He, well, that's how I find this bug. We weren't passing the `subscription_url` to this notification 😄